daliuge-engine/test/test_drop_ports.py

```diff
@@ -68,7 +68,7 @@ def _create_and_run_graph_spec_from_lgt(self, logical_graph_name: str):
         roots = graph_loader.createGraphFromDropSpecList(appDropSpec)
         # drops = [v for d,v in drops.items()]
         leafs = droputils.getLeafNodes(roots)  
-        with  droputils.DROPWaiterCtx(self, leafs, timeout=3):
+        with  droputils.DROPWaiterCtx(self, leafs, timeout=600):
```

Comment:
Does the timeout really need to be that long? If, for some reason this gets into a hanging state, the test suite will sit there for 10 min before it continues. I had that a few times through vscode and it is really not helpful.

Reply:
It absolutely does not - I did this for debugging purposes but accidentally committed the edit. I will revert these changes.